pmlc/rt/prng.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Intel Corporation
+#include "mlir/ExecutionEngine/RunnerUtils.h"
+#include "pmlc/compiler/registry.h"
+#include "pmlc/util/logging.h"
+
+extern "C" void
+plaidml_rt_prng(size_t stateRank, UnrankedMemRefType<float> *state,
```

Comment:
`state` and `newState` should be `UnrankedMemRefType<int32_t>`

Reply:
Resolved